src/test/test-sleep.c

```diff
@@ -26,10 +27,8 @@ static int test_fiemap(const char *path) {
         if (fd < 0)
                 return log_error_errno(errno, "failed to open %s: %m", path);
         r = read_fiemap(fd, &fiemap);
-        if (r == -EOPNOTSUPP) {
-                log_info("Skipping test, not supported");
-                exit(EXIT_TEST_SKIP);
-        }
+        if (r == -EOPNOTSUPP)
+                return log_tests_skipped("Not supported");
```

Comment:
Needs to check the return value in main().

Reply:
Fixed.